tutorials/nservicebus-sagas/2-timeouts/tutorial.md

```diff
@@ -70,7 +72,7 @@ NOTE: This tutorial uses 20 seconds as a timeout value for simplicity. In produc
 
 Besides the `context`, the `RequestTimeout` method has two parameters of note. One is the `TimeSpan` which tells us how long to wait before sending our trigger message. In this case, it's 20 seconds.
 
-NOTE: Instead of a `TimeSpan`, we could provide a `DateTime` instance, such as `DateTime.UtcNow.AddDays(10)`. When using this form, remember to use UTC dates to avoid issues arising from Daylight Savings Time.
+NOTE: Instead of a `TimeSpan`, we could provide a `DateTime` instance, such as `DateTime.UtcNow.AddDays(10)`. When using this form, make sure if you want to pass a local time which is affected by Daylight Savings Time (DST) changes or to use UTC dates to avoid DST conversion issues.
```

Comment:
```suggestion
NOTE: Instead of a `TimeSpan`, we could provide a `DateTime` instance, such as `DateTime.UtcNow.AddDays(10)`. When using this form, remember that local time is affected by Daylight Savings Time (DST) changes: use UTC dates to avoid DST conversion issues.
```